examples/prelude.nim

```diff
@@ -1,23 +1,10 @@
-(macro (#a ; #b) #b)
-; ( (rassoc 1 ;)
-  ; macro
-      (
-        #a
-        ----------
-        #b
-      )
-      (macro #a #b)
-  ; rassoc 2 ----------
-  ; rassoc 3 //
-  ; // we now have comments
-
-  ; import #module
-    ----------
-    bash (cat #module)
-
-  ; // another syntax for macros
-  ; rassoc 2 => ;
-  ; macro (#a => #b) (macro #a #b)
-
-  ; import examples/math.nim
+((macro (defined ; #b) #b)
+  ; ( (rassoc 1 ;)
+    ; macro (#a => #b) (macro #a #b)
+    ; (() ; #a) => #a
+    ; rassoc 2 =>
+    ; import #module => bash (cat #module)
+    ; import examples/math.nim
+    ; math (1 + 2)
```

Comment:
weird thing to have in prelude ;)

Reply:
yeah, what the heck is that thing doing in here? I'll get rid of him ASAP